vignettes/notame_normalization_and_feature_selection.qmd

```diff
@@ -0,0 +1,298 @@
+
+
+# Setup
+Let's attach the *notame* and load the preprocessed data as a SummarizedExperiment object returned from MSExperiment::quantify().
+
+```{r, message=FALSE, warning=FALSE}
+library(doParallel)
+library(cowplot)
+library(limma)
+library(notame)
+library(SummarizedExperiment)
+
+registerDoParallel(cores = 14)
+
+res <- readRDS("res.RDS")
+```
+
+The SummarizedExperiment container used for the data is very similar to the 
+MetaboSet container used in *notame*. The MetaboSet container is derived from the widely used ExpressionSet container, and is extended with three slots which can store the names of the columns for study group, subject identifier and time point. The accessors are a bit different from SummarizedExperiment; use exprs() to access the abundances, fData() to access feature metadata and pData() to access sample metadata. The sample and feature metadata needs to be modified somewhat for the MetaboSet container. 
+
+```{r, message=FALSE, warning=FALSE}
+pheno_data <- colData(res)
+feature_data <- rowData(res)
+
+# Rename columns of sample metadata
+rename_ind <- which(names(pheno_data) %in% 
+                    c("sample_name", "sample_type", "injection_index"))
+names(pheno_data)[rename_ind] <- c("Sample_ID", "QC", "Injection_order")
+
+# Match the rownames of sample metadata with Sample_ID
+rownames(pheno_data) <- pheno_data$Sample_ID
+
+# Change name of pooled samples to "QC" in the "QC" column
+pheno_data$QC[pheno_data$QC == "pool"] <- "QC"
+
+# Convert phenotype column to factor
+pheno_data$phenotype %<>% as.factor()
+
+# Create Feature_ID column in feature metadata
+feature_data <- data.frame(Feature_ID = rownames(feature_data), feature_data)
+
+# Create "Split" column in feature data with analytical mode
+feature_data$Split <- "HILIC_pos"
+
+# Construct MetaboSet
+res <- construct_metabosets(exprs = assay(res), 
+                            pheno_data = as.data.frame(pheno_data), 
+                            feature_data = as.data.frame(feature_data),
+                            group_col = "phenotype", split_data = FALSE)
+```
+
+# Normalization
+Features with a low detection rate are flagged, as they are deemed too unreliable not only for statistical analysis, but also for the normalization process which relies heavily on QC samples. Flagged features are automatically excluded in *notame*, but can be included using `all_features = TRUE` in many functions. We set the detection rate threshold for QC samples at 70% as per Broadhurst et al. 2018 [@qcguidelines], plus a within-group threshold of 80%. The within-group threshold must be met in at least one study group.
```

Comment:
I can see you are quotting a paper, do you have a `reference.bib` file for this vignette ? if yes can you also add it to the github repo ? otherwise no worries I'll build it later on myself. if you add it maybe name it `reference_notame.bib`